src/client/core/index.js

```diff
@@ -1,14 +1,15 @@
 import hammerhead from './deps/hammerhead';
 
-import $ from './deps/jquery';
 import SETTINGS from './settings';
 import ERROR_TYPE from '../../reporter/errors/type';
 import COMMAND from '../../runner/test-run/command';
 import CROSS_DOMAIN_MESSAGES from './cross-domain-messages';
 import * as transport from './transport';
+import Sizzle  from './sandboxed-sizzle';
+import sizzleCustomSelectors  from './extensions/sizzle-custom-selectors';
```

Comment:
I suppose, `eslint` should raise an error here about of the unused `sizzleCustomSelectors` var. I think we should make this like the `exetendJQuerySelectors`
```js
import extendSizzleSelectors from '...';    // export default function
```